.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
```

Comment:
We should test multiple versions!!!

Reply:
```suggestion
        python-version: ['3.8', '3.10', '3.11', '3.12', '3']
```

Reply:
Ref": https://github.com/cylc/cylc-flow/blob/4aa492d95c82bc55ac2f49b7beeefb057fcae48c/.github/workflows/test_fast.yml